src/components/followAccount/index.js

```diff
@@ -0,0 +1,15 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { followedAccountAdded, followedAccountRemoved } from '../../actions/followedAccounts';
+import FollowAccount from './followAccount';
+
+const mapStateToProps = state => ({
+  accounts: state.followedAccounts.accounts,
+});
+
+const mapDispatchToProps = dispatch => ({
+  addAccount: data => dispatch(followedAccountAdded(data)),
+  removeAccount: data => dispatch(followedAccountRemoved(data)),
+});
```

Comment:
right now we are trying to use the mapDispatchToProps just as an object instead use the dispatch, could you changed or you need it in this way.

```
const mapDispatchToProps = {
  addAccount,
  removeAccount,
};
```

Reply:
done